Spigot-Server-Patches/0625-Add-sendOpLevel-API.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Mariell Hoversholm <proximyst@proximyst.com>
+Date: Tue, 29 Dec 2020 15:03:03 +0100
+Subject: [PATCH] Add sendOpLevel API
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index 58caa3240b90cdc661e1e32e3f5c312ed62c3c21..8362448aad77dff572f8e6fb30db1cb3ce953936 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -2100,6 +2100,16 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+             ? (org.bukkit.entity.Firework) entity.getBukkitEntity()
+             : null;
+     }
++
++    // This sends a packet with a status described here: https://wiki.vg/Entity_statuses#Player
++    private static final byte ENTITY_STATUS_PACKET_OPERATOR_LEVEL = 24;
++    @Override
++    public void sendOpLevel(byte level) {
++        Preconditions.checkArgument(level >= 0 && level <= 4, "Level must be within [0, 4]");
++
++        net.minecraft.server.Packet<?> packet = new net.minecraft.server.PacketPlayOutEntityStatus(this.getHandle(), (byte) (ENTITY_STATUS_PACKET_OPERATOR_LEVEL + level));
```

Comment:
PlayerList#a(EntityPlayer entityplayer, int i)

We'd likely be better off exposing that vs hard coding the packet out of bounds, otherwise the value may change etc and we wouldn't know, much easier to just expose existing logic, imho; Would just need a means to disable the perm recalc checks